tests/frontend/correspondence_generator/keypoint_aggregator/test_keypoint_aggregator.py

```diff
@@ -0,0 +1,117 @@
+"""Unit tests for keypoint aggregation for correspondence generators.
+
+Author: John Lambert
+"""
+from typing import Dict, Tuple
+
+import numpy as np
+
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.correspondence_generator.keypoint_aggregator.keypoint_aggregator_dedup import (
+    KeypointAggregatorDedup,
+)
+from gtsfm.frontend.correspondence_generator.keypoint_aggregator.keypoint_aggregator_unique import (
+    KeypointAggregatorUnique,
+)
+
+
+def test_keypoint_aggregator_unique_keypoints() -> None:
```

Comment:
could you separate these tests into different files? for purposes of consistency. 

And since this logic is tricky I think more tests for each class would be nice. For example: multiple non-unique keypoints, all keypoints are duplicates for an image, no duplicate keypoints in a call to the duplicate aggregator etc.

Reply:
Thanks, I re-used one test for two separate classes now, so we are up to 3 tests. If we split it apart, I would need to repeat that code, so I'd prefer we keep it consolidated in a single test.